source/payment_flow/index.html.md.erb

```diff
@@ -58,7 +59,7 @@ When your user reaches a page in your service where they need to make a payment,
 
 You should also have a clear call to action that tells your user they're about to pay. For example, a button that says __Pay now__ or __Continue to payment__.
 
-You do not need to tell your user that they'll be taken to GOV.UK Pay’s pages to make their payment.
+You do not need to tell your user that they'll be taken to GOV.UK Pay’s pages to make their payment. The following image shows an example payment page.
 
 <%= image_tag "/images/flow-service-payment-page.png", { :alt => '' } %>
```

Comment:
I believe we still need an alt tag for accessibility even if the text explains it.

Reply:
This has been checked with content accessibility team - so happy to approve.